server/start.js

```diff
@@ -0,0 +1,3 @@
+const app = require('./index.js');
+let port = 9000;
+app.listen(port, () => { console.log(`server listening on port ${port}`); });
```

Comment:
this shouldn't affect nodemon either, as it reads from 'main' in package.json which leads to start.js which imports our server index